tests/cairo_files/test_memory.cairo

```diff
@@ -108,18 +121,6 @@ func test__load__should_load_an_element_from_the_memory_with_offset{
     return ();
 }
 
-@external
-func test__load__should_fail__when_out_of_memory{
```

Comment:
what happened to this test? though I don't really understand it, there's no out of memory in the EVM right?

Reply:
yeah that's my understanding